src/ComposerRequireChecker/Cli/CheckCommand.php

```diff
@@ -116,6 +116,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $table = new Table($output);
         $table->setHeaders(['unknown symbol', 'guessed dependency']);
         $guesser = new DependencyGuesser($options);
+
+        natcasesort($unknownSymbols);
```

Comment:
Alternatively, perhaps also in the light of ease of testing, we could sort the symbols before returning them from [`LocateUsedSymbolsFromASTRoots::__invoke()`](https://github.com/maglnet/ComposerRequireChecker/blob/bf9d08ec45b06ff6f090039c33ac9047750e1a59/src/ComposerRequireChecker/UsedSymbolsLocator/LocateUsedSymbolsFromASTRoots.php#L33).

What do you think?

Reply:
If the current sorting is based on how locale/filesystem are configured, then sorting in the dependency would be perfectly OK 👍

Reply:
@Ocramius 

Apologies, not sure I understand at the moment. Do you prefer to 

1. leave sorting in the `CheckCommand`
2. move sorting to [`LocateUsedSymbolsFromASTRoots::__invoke()`](https://github.com/maglnet/ComposerRequireChecker/blob/bf9d08ec45b06ff6f090039c33ac9047750e1a59/src/ComposerRequireChecker/UsedSymbolsLocator/LocateUsedSymbolsFromASTRoots.php#L33)

?

Reply:
I'd say that moving it to `LocateUsedSymbolsFromASTRoots` would be nice :+1:

Reply:
@Ocramius 

Adjusted!